hyperspy/_signals/eds.py

```diff
@@ -233,109 +233,6 @@ def rebin(self, new_shape):
                     *= factor
         return s
 
-    def linear_bin(self, scale):
```

Comment:
If you mean, is an EDS signal able to call the method - after moving it to BaseSignal from EDS - then yes :)